tools/travis/osx_install.sh

```diff
@@ -3,7 +3,7 @@ set -ex
 
 brew update
 brew install ccache
-brew tap caskroom/cask
+brew tap homebrew/cask-cask
```

Comment:
```suggestion

```
From brew [homepage](https://brew.sh/), it looks like cask is built in... Try to remove this line...

Reply:
This fix works, I tested it in #4218 ;-)